docs/developer/architecture.rst

```diff
@@ -254,18 +254,8 @@ API
 The API is the outer-most layer that faces users. The following classes contain Modin's implementation of the pandas API:
 
 .. toctree::
-   :caption: Base Pandas Dataset API
-
    /flow/modin/pandas/base
-
-.. toctree::
-   :caption: modin.pandas.DataFrame API
-
    /flow/modin/pandas/dataframe
-
-.. toctree::
-   :caption: modin.pandas.Series API
-
    /flow/modin/pandas/series
```

Comment:
pls make sure this renders correctly

Reply:
[this particular part](https://modin--3640.org.readthedocs.build/en/3640/developer/architecture.html#api), as well as the whole [architecture page](https://modin--3640.org.readthedocs.build/en/3640/developer/architecture.html) seems to be rendered correctly